src/redist/targets/GenerateLayout.targets

```diff
@@ -466,6 +466,12 @@
                       Overwrite="true" />
   </Target>
 
+  <Target Name="LayoutWorkloadUserLocalMarker">
```

Comment:
I think the file should only be written when building with source-build.  i.e. `Condition=" '$(DotNetBuildFromSource)' == 'true' "`